packages/components/src/components/Alert.tsx

```diff
@@ -10,23 +10,32 @@ export const Alert: React.FC<{
   backgroundColor: string;
   headingColor: string;
   bodyColor: string;
-  icon: React.ReactNode;
-  children: React.ReactNode;
+  icon: (props: React.ComponentProps<"svg">) => JSX.Element;
+  iconColor: string;
+  children?: React.ReactNode;
 }> = ({
   heading = "Error",
   backgroundColor,
   headingColor,
   bodyColor,
-  icon,
+  icon: Icon,
+  iconColor,
   children,
 }) => {
   return (
     <div className={`rounded-md p-4 ${backgroundColor}`}>
       <div className="flex">
-        <div className="flex-shrink-0">{icon}</div>
+        <Icon
+          className={`h-5 w-5 flex-shrink-0 ${iconColor}`}
+          aria-hidden="true"
+        />
         <div className="ml-3">
-          <h3 className={`text-sm font-medium ${headingColor}`}>{heading}</h3>
```

Comment:
Rationale for replacing `h3` with `header`, here and in other files: lib users can have their own header hierarchy, and we can't predict how deeply in their tree the SquigglePlayground (or whatever) is used. It's a bad idea to insert `h3` under someone else's `h4`.

Not that anyone cares about clean semantic HTML in 2022, of course... except for people with screen readers, but squiggle charts won't be much use for them anyway.

Reply:
Good catch!